src/CacheManager.Serialization.Json/project.json

```diff
@@ -33,9 +33,17 @@
     "Newtonsoft.Json": "6.0.8"
   },
   "frameworks": {
-    "dnx451": { },
+    "dnx451": {
+      "dependencies": {
+        "System.IO.Compression": "4.1.0-beta-23516"
```

Comment:
I think for dnx451 and net45, you should not add .net core assemblies. Either the Compression stuff is already available, or you add them to `frameworkAssemblies`.